test/1.frontend/2.login.test.js

```diff
@@ -243,17 +256,6 @@ describe('Accessing with login', function() {
     });
 });
 
-describe('Login out', function() {
-    describe('GET /logout', function() {
-        it('should redirect to login', function(done) {
-            agent
-                .get('/logout')
-                .expect('location', '/login')
-                .expect(302, done);
-        });
-    });
-});
-
```

Comment:
I think we should do POST logout here, or else next test should fail and do not redirect you to login page, shouldn't it?

Reply:
Right, just rechecked it is affecting the previous login testing.  Thanks :)